README.md

```diff
@@ -93,6 +93,22 @@ Diagrams are generated "automatically" by ???
 
 ## User Guide
 
+###Database Management
+To run a local version of this project you will need to start a 
+repl and initialize the vanilla_db.
+This can be done in 
+````$xslt
+src/clj/vanilla/db/core.clj
+````
+In the first comment block.
```

Comment:
this only needs to be 3 back-ticks before and after, and I don't know where the $xslt came from

Reply:
This was an autofill thing intellij put in when I started typing the back-ticks. I have cleaned it up to match the rest of the readme on my latest push